src/GitElephant/Command/CloneCommand.php

```diff
@@ -59,6 +63,31 @@ public function cloneUrl($url, $to = null)
             $this->addCommandSubject2($to);
         }
 
+        $binaryVersion = '2.0.0.0';
```

Comment:
I suppose this is a relict from testing? Should we not compare with the actual version?

Reply:
yes, totally.. I left this in on accident. Thanks for checking. I did this to run through phpunit tests until #147 gets merged in. Will remove.